vrm.ts

```diff
@@ -290,27 +442,27 @@ export class Vrm extends Component {
         };
     }
 
-    _findAndParseNodeConstraints(extensions) {
-        const traverse = (object) => {
+    private _findAndParseNodeConstraints(extensions: any, idMapping: number[]): void {
+        const traverse = (object: Object3D) => {
             const nodeExtensions = extensions.node[object.objectId];
             if (nodeExtensions && 'VRMC_node_constraint' in nodeExtensions) {
                 const nodeConstraintExtension = nodeExtensions['VRMC_node_constraint'];
 
                 const constraint = nodeConstraintExtension.constraint;
-                let type, axis;
+                let type: string | undefined, axis: [number, number, number] | undefined;
                 if ('roll' in constraint) {
                     type = 'roll';
-                    axis = VRM_ROLL_AXES[constraint.roll.rollAxis];
+                    axis = VRM_ROLL_AXES[constraint.roll.rollAxis as string];
```

Comment:
Here you could do `as keyof VRM_ROLL_AXES` I think, but you might need a `as unknown` first